README.adoc

```diff
@@ -22,87 +22,69 @@ This section describes some important information about how this repository is s
 
 - The component name, version, and start page are configured in each branch's `antora.yml` file.
 - The navigation for all modules is stored in the ROOT module's `nav.adoc` file.
-- The {url-playbook}[docs site playbook] instructs Antora to automatically build the site using content in the `master` branch as well as any tags that are prefixed with *v*.
+- The {url-playbook}[docs site playbook] instructs Antora to automatically build the site using content in the `master` branch as well as any branches that are prefixed with `v/`.
 
 == Release Workflow
 
-Documentation for new releases is hosted in versioned tags that are prefixed with `v`. For example, content for the `4.1.1` version is hosted in the `v4.1.1` tag. The `latest-dev` content is stored in the `master` branch
+Documentation for new releases is hosted in versioned branches that are prefixed with `v/`.
 
-The documentation build process is triggered whenever you push a new tag to this repository or commit to the `master` branch.
+We support documentation for the latest patch releases of minor versions. For example, content for the `4.1` version is hosted in the `v/4.1` branch. This branch contains content for the latest patch release of version 4.1.
+
+The `latest-dev` content (snapshot content) is stored in the `master` branch.
+
+The documentation build process is triggered whenever you create a new branch with the `v/` prefix, push to an existing `v/` branch, or push to the `master` branch.
 
 === Patch Releases
 
 This section guides you through the steps for releasing documentation for patch releases.
 
-. Find the `.z` branch for your version. For example, if you're working on content for version 4.1.2, find the branch named `4.1.z`.
+. If you are releasing a new `latest` version, submit a pull request to the `develop` branch of the link:{url-playbook}[`hazelcast/hazelcast-docs`] repository to do the following:
```

Comment:
I check the content of antora file on current `v/4.1` branch and there is version which includes also patch part - https://github.com/hazelcast/imdg-docs/blob/v/4.1/docs/antora.yml#L3 - I think this should be updated during patch release - please add this step here as well.